bool createMap
int32 roiIndex
bool deleteMap
---
